src/platforms/aarch64_linux_gnu.rs

```diff
@@ -0,0 +1,64 @@
+//!
+//! The zkEVM LLVM `linux-gnu` builder.
+//! Exact copy of x86_64_linux_gnu
```

Comment:
not sure if there's more elegant way to not copy-paste from x86_64_linux_gnu

Reply:
No worries.
I can merge it later.

Let's run this branch from your workflow.

Reply:
I have no idea yet if we can build LLVM with the same script :)

Reply:
it builds, i already tested it